plex-service/src/main/java/com/cleverpine/plex/mapper/EpisodeMapper.java

```diff
@@ -0,0 +1,81 @@
+package com.cleverpine.plex.mapper;
+
+import com.cleverpine.plex.dto.EpisodeDto;
+import com.cleverpine.plex.entity.future.EpisodeEntity;
+import com.cleverpine.plex.model.EpisodeListItem;
+import com.cleverpine.plex.model.EpisodeListResponse;
+import com.cleverpine.plex.model.SingleEpisode;
+import com.cleverpine.plex.model.SingleEpisodeResponse;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+import java.util.List;
+
+@Mapper
```

Comment:
Use @Mapper(componentModel = "spring") instead, that way you don't need to instantiate the mapper on every use.